.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Maybe I would prefer https://github.com/feross/standard as there is no possible discussions because it is as it is.